Sources/SegmentFirebase/FirebaseDestination.swift

```diff
@@ -90,11 +90,16 @@ public class FirebaseDestination: DestinationPlugin {
     public func track(event: TrackEvent) -> TrackEvent? {
         
         let name = formatFirebaseEventNames(event.event)
-        var parameters: [String: Any]? = nil
+        var parameters: [String: Any] = [:]
```

Comment:
I don't think this is a good idea. Do we know if the `logEvent` call behaves differently if you pass `nil` vs an empty dict?